python/Makefile

```diff
@@ -0,0 +1,12 @@
+IMAGE_NAME=ondewo-python-proto-compiler
+
+build:
+	docker build -t ${IMAGE_NAME} .
+
+run_example:
+	docker run -v ${shell pwd}:/opt/ondewo-proto-compiler -it -e PROTOS_DIR=example/protos -e APIS_DIR=example -e OUTPUT_FOLDER=. ${IMAGE_NAME}
```

Comment:
Can't this be done easier?

Like..
We can tell the user to put their protos in the `protos` folder and then COPY that inside the Docker Image.
Additionally, we could map a volume out for the output so the user doesn't need to specify the input and output folder.